backend/.env.example

```diff
@@ -1 +1,9 @@
 HUGGING_FACE_API_KEY=HF API KEY
+DATABASE_URI="postgresql://<username>:<password>@<host>/<database_name>"
+BACKEND_CORS_ORIGINS=["http://localhost", "http://localhost:4200", "http://localhost:3000", "http://localhost:8080", "https://localhost", "https://localhost:4200", "https://localhost:3000", "https://localhost:8080", "http://dev.oasst.laion.ai", "https://stag.oasst.laion.ai", "https://oasst.laion.ai"]
+REDIS_HOST=localhost
+REDIS_PORT=6379
+
+
+export DEBUG_SKIP_EMBEDDING_COMPUTATION=False
+export DEBUG_SKIP_TOXICITY_CALCULATION=False
```

Comment:
Thanks for adding all these! This is much cleaner than having the examples in the readme